annotations/src/main/java/com/statful/client/annotations/Timer.java

```diff
@@ -38,4 +38,9 @@
      * @return An array of aggregations to send to Statful
      */
     Aggregation[] aggregations() default {};
+
+    /**
+     * @return An int of the sampleRate to send to Statful
+     */
+    int sampleRate() default 0;
```

Comment:
Actually on BufferedMetricsSender there's a constant MIN_SAMPLE_RATE defined as 1. If the value is lower, the metric is sent with a sample rate 1 (as defined in sanitizeSampleRate function). So I think the enable attribute actually makes some sense, right?